### PR TITLE
CLID-66,CLID-80: changes README.md

### PR DESCRIPTION
**Pull request by aguidirh**
# Description

This PR updates the v2 README.md and move some docs to v2 folder.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Check the README.md under v2 folder.

## Expected Outcome
The content is different from the one in main branch.

### CODE REVIEW

File: v2/README.md [hunk 1]
```diff
@@ -1,671 +1,154 @@
...
+## Development
+This section will give an overview about oc-mirror v2 architecture. Pull requests are always welcome in the oc-mirror repository.
 
-Typically there should be entries for all the registries found in the policy.json file as an example for registry.redhat.io
+### Architecture
+![Alt text](./assets/architecture.png)
 
-A file for each entry found in the policy.json
+### Image Set Configuration Specs
+[Here](https://github.com/openshift/oc-mirror/blob/e7889a7ec70dd66b0d6a7ba6dedc3e4b93ebf4de/v2/pkg/api/v2alpha1/type_config.go#L16) it is possible to find the specification of the Image Set Configuration with all available fields.
```
Comment (lmzuccarelli) [inline v2/README.md:132]:
```suggestion
[Link](https://github.com/openshift/oc-mirror/blob/e7889a7ec70dd66b0d6a7ba6dedc3e4b93ebf4de/v2/pkg/api/v2alpha1/type_config.go#L16) that shows details about the specification of the Image Set Configuration with all available fields.
```

File: v2/README.md [hunk 2]
```diff
@@ -1,671 +1,154 @@
...
-2023/03/24 15:01:05  [DEBUG]  : destination docker://localhost.localdomain:5000/testlmz/working-dir/test-lmz/redhat-operator-index/v4.12/aws-load-balancer-operator.v0.2.0/openshift4 
-2023/03/24 15:01:05  [DEBUG]  : source oci:working-dir/test-lmz/additional-images/ubi9/ubi 
-2023/03/24 15:01:05  [DEBUG]  : destination docker://localhost.localdomain:5000/testlmz/working-dir/test-lmz/ubi9 
-2023/03/24 15:01:05  [INFO]   : completed batch 0
-
-
+make test-integration
 ```
 
+## To Know More About
```
Comment (lmzuccarelli) [inline v2/README.md:148]:
```suggestion
## Find out more
```

Comment (aguidirh) [reply v2/README.md]: Done ✅ 

Thanks for the suggestion.

File: v2/README.md (hunk 1 above)
Comment (aguidirh) [reply v2/README.md]: Done ✅ 

Thanks for the suggestion.

**Pull request merged**